web/src/main/java/org/cbioportal/web/parameter/sort/PatientSortBy.java

```diff
@@ -0,0 +1,7 @@
+package org.cbioportal.web.parameter.sort;
+
+public enum PatientSortBy {
+
+    stableId,
```

Comment:
I think it is convention to have enum constants be all upper case. But maybe for the framework to work correctly they must match our data member names.